src/main/java/org/opensearch/performanceanalyzer/decisionmaker/deciders/searchbackpressure/SearchBackPressurePolicy.java

```diff
@@ -103,14 +103,18 @@ public SearchBackPressurePolicy(SearchBackPressureClusterRCA searchBackPressureC
      * @param issue an issue with the application
      */
     private void record(HotResourceSummary summary) {
+        LOG.debug("SearchBackPressurePolicy capturing resource summary: {}", summary);
```

Comment:
This is a `DEBUG` level log which will be only logged if the log level is DEBUG. We use debug for devleopment purposes not prod. I think by default the log level is `INFO` so this would not be logged.

Reply:
Don't agree that we don't use DEBUG in prod. In OpenSearch, we have a places where debug is enabled by default like for actions. And sometimes we enable it for specific debugging purposes.
Dumping whole objects seems unnecessary as we may or may not require all fields. So better to log specific stuff whatever is needed.

Reply:
There is no logic to change the logging levels dynamically in RCA but It makes sense to avoid unnecessary details in logs. Changing it to trace,